docs/manual/index-checker.tex

```diff
@@ -303,23 +303,27 @@ \section{Array minimum lengths\label{index-minlen}}
 annotated as
 \refqualclasswithparams{checker/index/qual}{LTLengthOf}{"arr"}.
 
+The MinLen type system is implemented in practice by the Constant Value Checker,
```

Comment:
MinLen is no longer a type system. Say something like, `@MinLen` is checked by the CVC.  Then maybe add something about how the other two array annotations can be used, too.  Maybe add an example using `@ArrayLen`.